### PR TITLE
Update changes to code post code review feedback.

### PR DESCRIPTION
**Pull request by jrai0792**
### Sope of the project 
Building an HTML page that matches the appearance of the ### youtube video player page.

### Project Summary
- Added blank index.html file.

- Added navigation bar with its elements.

- Added main video and its elements.

- Added recommended video and elements.

- Added CSS file and styled all the sections.

### CODE REVIEW

**Comment by oracleot** [comment]
I verified the pull request with the added blank index.html file

File: index.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+      <ul class="like-dislike-share color-gray">
+        <li>
+          <i class="fa fa-thumbs-o-up" aria-hidden="true"></i>&ensp;8&emsp;
+        </li>
+        <li>
+          <i class="fa fa-thumbs-o-down" aria-hidden="true"></i>&ensp;4&emsp;
+        </li>
+        <li>
+          <i class="fa fa-share" aria-hidden="true"></i>&ensp;Share&emsp;
```
Comment (HADDADSOHAIB) [inline index.html]: The share icon should be a button

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change required :recycle: 

### Hi @jrai0792 and @oracleot :hand: 

### Great job with the project, you did great work. There is some changes to be made to approve the project:
- check my comments for additional requirements.
- In the project description, there was a link that you should have used to build your project:
image
http://archive.vn/Bss88
there a slight different between your project and that link in the section under the main video.

 | Source Page	| Clone Page |
|:---:|:---:|
|Screenshot from 2020-07-10 11-51-12 |Screenshot from 2020-07-10 11-50-59 |

- A pull request description should have information about what this pull request add to the project.
Screenshot from 2020-07-10 12-01-48
Please improve the current description

### Once done with the changes, ask for another code review.

File: index.html (hunk 1 above)
Comment (oracleot) [reply index.html]: Thanks for the code review, we will resolve ASAP and ask for another review

Comment (jrai0792) [reply index.html]: Code revamped based on the feedback from the code review.

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQIRED :recycle: :stop_button: 
### HELLO @jrai0792 AND @oracleot AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT. 
### YOUR PROJECT LOOKS AMAZING :clap:  VERY NICE WORK :partying_face: HOWEVER, THERE ARE A FEW THINGS THAT HAVE PREVENTED ME FROM APPROVING THIS PROJECT KINDLY CHECK BELOW :arrow_down: 

- [x] Just like stated above you need to make sure you have a descriptive pull request message. You can do this by adding a title and a summary of what you have worked on using bullet points.

- [x] Also you do have a readme file :clap: but let us make it more descriptive by adding more information to it. For example, you can add a screenshot of your project.

### when you are done improving on this kindly submit your project for another review. 
### good work working on the changes that were requested by the previous TSE :clap: 

### HAPPY CODING :computer:

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1: :1st_place_medal: 
### VERY IMPRESSIVE WORK YOUR PROJECT IS NOW APPROVED .. GOOD WORK IMPROVING AS REQUESTED :clap: 

[OPTIONAL]
your HTML is showing some warnings when validated by this  [tool](https://validator.w3.org/nu/#textarea) you guys can try and fix the warnings. However, since they are not errors this is not a requirement

### HAPPY CODING :computer:

**Comment by oracleot** [comment]
Thanks @peterrobert!

**Pull request merged**